benchmarks/bench_kn_nlearners.py

```diff
@@ -0,0 +1,311 @@
+# pylint: disable=missing-function-docstring
+import argparse
+import numpy as np
+import trueskill
+
+from truelearn import learning, datasets
+
+# fmt: off
+def _erfc(x):
```

Comment:
I've only provided a one-line description for this because 1) it's a mathematical formula and 2) we don't necessarily need to document this (it's private and self-explanatory).